.gitignore

```diff
@@ -52,4 +52,8 @@ xcuserdata/
 
 # End of https://www.gitignore.io/api/swift,xcode,cocoapods
 
-.nogit
+### TAGS, emacs, and editor files
+TAGS
+todo.org
```

Comment:
I would keep the TAGS and todo.org files in some local gitignore. Though they should not be in the repository, to put every specific dev environment into account is not the job of a gitignore.